internal/cli/picker_options.go

```diff
@@ -0,0 +1,32 @@
+package cli
+
+type pickerOptions []pickerOption
```

Comment:
The previous iteration using `maps` had an undesirable behavior that options get randomized on every run. Formalizing this data structure to a list to have a deterministic order -- but also having methods on the list simplifies the call sites.